taqueria-vscode-extension/package.json

```diff
@@ -310,14 +310,14 @@
 			{
 				"command": "taqueria.add_contract",
 				"category": "Taqueria",
-				"title": "Add contract to registry",
+				"title": " Add contract to registry",
```

Comment:
Because the space is added before both `Add` and `Remove` commands, they will be sorted in the same spot no matter what letter other commands start with (as long as none of the others starts with a space)

The shift is not noticeable, as it's in the tooltip.

Reply:
To be honest, for most of the changes at least a video is needed to see the change

Reply:
ahh ok ok

Reply:
it may seem like too much work in that case